docker/inference/dockerfile.ctr

```
@@ -1,10 +1,15 @@
 # syntax=docker/dockerfile:1.2
 ARG TRITON_VERSION=22.02
-ARG IMAGE=nvcr.io/nvidia/tritonserver:${TRITON_VERSION}-py3-min
+#ARG TRITON_DISTRIUBTION=py3
+ARG TRITON_DISTRIUBTION=py3-min
+ARG IMAGE=nvcr.io/nvidia/tritonserver:${TRITON_VERSION}-${TRITON_DISTRIUBTION}
```

nitpick: distribution is misspelt - and also lets get rid of commented out code:

```suggestion
ARG TRITON_DISTRIBUTION=py3-min
ARG IMAGE=nvcr.io/nvidia/tritonserver:${TRITON_VERSION}-${TRITON_DISTRIBUTION}
```